tools/build.py

```diff
@@ -53,6 +57,18 @@ def main():
                 removed += 1
                 print("  Removing obsolete file => " + file)
                 os.remove(file)
+
+    if platform.system() == "Windows":
+        work_drive = "P:\\"
```

Comment:
Work drive needs to be configurable. It's not P:\\ on build server.

Reply:
make.cfg has `work_drive` entry.

Reply:
so read from make.cfg?

Reply:
yup. Maybe we want a seperate entry for linux with armake?

Reply:
I've done something for another project.
I'll add this here too, if I find some time.